start.sh

```diff
@@ -53,33 +48,28 @@ docker-compose --compatibility up -d grafana
 echo "Sleeping for $SLEEP_TIME secs to give Grafana some time to start up..."
 sleep $SLEEP_TIME
 
-existingPerfanaUserId=$(curl -X GET -H "Content-Type: application/json"  http://$GRAFANA_CREDS@localhost:3000/api/auth/keys 2>/dev/null | jq '.[] | select(.name == "Perfana")' | jq .id)
-
-if [ -z "$existingPerfanaUserId" ];  then
-    echo "Creating Grafana API key..."
-    apiKey=$(curl -X POST -H "Content-Type: application/json" -d '{"name":"Perfana","role":"Admin"}' http://$GRAFANA_CREDS@localhost:3000/api/auth/keys 2>/dev/null | jq -r '.key')
-    if [[ -z "$apiKey" ]] || [[ "$apiKey" == "null" ]]; then
-        echo "Issue creating Grafana API key, no API key returned from create call: abort!"
-        exit 123
-    fi
-    echo "Replacing apiKey in docker-compose file with: ${apiKey}"
-    if [ "$(check-os)" == "mac" ]; then
-        sed -i '' "s/\"apiKey\": \".*\"/\"apiKey\": \"$apiKey\"/g" docker-compose.yml
-    else
-        sed -i "s/\"apiKey\": \".*\"/\"apiKey\": \"$apiKey\"/g" docker-compose.yml
-    fi
+perfanaUser=`curl -X GET -H "Content-Type: application/json" "http://$GRAFANA_CREDS@localhost:3000/api/auth/keys" 2>/dev/null`
+echo "perfanaUser: $perfanaUser"
+if [[ $perfanaUser != *"Perfana"* || $perfanaUser != *"Admin"* ]]; then
+  echo "No Perfana user found. Creating new Grafana API key."
```

Comment:
Waarom niet "No Perfana api key found ..."

Reply:
Because it is the user that is fetched. Not the api key.